pkgs/development/python-modules/pydrive2/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib,
+  python3,
+  fetchPypi,
+  google-api-python-client,
+  oauth2client,
+  pyopenssl,
+  pyyaml,
+  six,
+ }:
+
+python3.pkgs.buildPythonPackage rec {
```

Comment:
```suggestion
buildPythonPackage rec {
```

Reply:
fixed in decb45313390f19369ceeea3d34700add2c10700